include/fastrtps/rtps/network/NetworkFactory.h

```diff
@@ -1,20 +0,0 @@
-// Copyright 2016 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
Please, be aware that this is not the proper `NetworkFactory` header file but only a "link" to the proper header. You should also modify the `fastdds/rtps/network` folder. Take this into account for the other headers too.

Reply:
According to what we discussed internally, this file should no longer exists here